mopidy/mpd/translator.py

```diff
@@ -58,26 +58,37 @@ def track_to_mpd_format(track, position=None, stream_title=None):
         result.append(('Id', tlid))
     if track.album is not None and track.album.musicbrainz_id is not None:
         result.append(('MUSICBRAINZ_ALBUMID', track.album.musicbrainz_id))
-    # FIXME don't use first and best artist?
-    # FIXME don't duplicate following code?
+
     if track.album is not None and track.album.artists:
-        artists = artists_to_mpd_format(track.album.artists)
-        result.append(('AlbumArtist', artists))
-        artists = [
-            a for a in track.album.artists if a.musicbrainz_id is not None]
-        if artists:
-            result.append(
-                ('MUSICBRAINZ_ALBUMARTISTID', artists[0].musicbrainz_id))
+        result.append(
+            ('AlbumArtist',
+                concatenate_multiple_values(track.album.artists, 'name')))
+        musicbrainz_ids = concatenate_multiple_values(
+            track.album.artists, 'musicbrainz_id')
+        if musicbrainz_ids:
+            result.append(('MUSICBRAINZ_ALBUMARTISTID', musicbrainz_ids))
+
     if track.artists:
-        artists = [a for a in track.artists if a.musicbrainz_id is not None]
-        if artists:
-            result.append(('MUSICBRAINZ_ARTISTID', artists[0].musicbrainz_id))
+        musicbrainz_ids = concatenate_multiple_values(
+            track.artists, 'musicbrainz_id')
+        if musicbrainz_ids:
+            result.append(('MUSICBRAINZ_ARTISTID', musicbrainz_ids))
 
     if track.composers:
-        result.append(('Composer', artists_to_mpd_format(track.composers)))
+        result.append(
+            (
+                'Composer',
+                concatenate_multiple_values(track.composers, 'name')
+            )
+        )
 
     if track.performers:
-        result.append(('Performer', artists_to_mpd_format(track.performers)))
+        result.append(
+            (
```

Comment:
Yeah sure, this was my first experience with flake8, and with formatting to 80 characters. flake8 didn't like 2 parens on the same line, or something about the first way I did it anyway. I think renaming the function sounds like a good option.